src/ui/map.js

```diff
@@ -1528,7 +1525,7 @@ class Map extends Camera {
         // Even though `_styleDirty` and `_sourcesDirty` are reset in this
         // method, synchronous events fired during Style#update or
         // Style#_updateSources could have caused them to be set again.
-        if (this._sourcesDirty || this._repaint || this._styleDirty || this._placementDirty || this.isEasing()) {
+        if (this._sourcesDirty || this._repaint || this._styleDirty || this._placementDirty) {
```

Comment:
If we're planning to fully convert the handlers to use render-loop-driven camera, then we should:

* Have handlers call `map._update` explicitly when they need to trigger a render, as in #6068
* Remove the `move`/`zoom` => `this._update` bindings
* Restore this `this.isEasing()` trigger